BraveRewardsUI/QA Settings/QASettingsViewController.swift

```diff
@@ -4,207 +4,139 @@
 
 import UIKit
 import BraveRewards
+import BraveShared
+import Shared
+import Static
+
+private typealias EnvironmentOverride = Preferences.Rewards.EnvironmentOverride
 
 /// A special QA settings menu that allows them to adjust debug rewards features
-public class QASettingsViewController: UIViewController {
+public class QASettingsViewController: TableViewController, UITextFieldDelegate {
   
   public let rewards: BraveRewards
   
   public init(rewards: BraveRewards) {
     self.rewards = rewards
-    super.init(nibName: nil, bundle: nil)
+    super.init(style: .grouped)
   }
   
   @available(*, unavailable)
   required init(coder: NSCoder) {
     fatalError()
   }
   
+  private let segmentedControl = UISegmentedControl(items: EnvironmentOverride.allCases.map { $0.name })
+  
+  @objc private func environmentChanged() {
+    let value = segmentedControl.selectedSegmentIndex
+    guard let _ = EnvironmentOverride(rawValue: value) else { return }
+    Preferences.Rewards.environmentOverride.value = value
+    self.rewards.reset()
+    self.showResetRewardsAlert()
+  }
+  
+  private let reconcileTimeTextField = UITextField().then {
```

Comment:
What about setting keyboard type to numeric? Do we always set numbers to this field?

Reply:
Yes, but then I'd have to add a done button in an accessory toolbar to do that and it's just a QA menu anyways :P